### ISSUE TITLE
Incorrect Wording

### ISSUE THREAD

**Issue by FelixDGG** [KhronosGroup/3DC-Asset-Creation#1]
In section sec01_FileFormatsAndAssetStructure the base asset is called "source asset".

In section sec04_UVCoordinates the base asset is called "source model".

In the summary it is only referred to "Source Model" (https://github.com/KhronosGroup/3DC-Asset-Creation/blob/main/asset-creation-guidelines/RealtimeAssetCreationGuidelines.md).

From discussion in the group and the text under "Publishing Target", the name should be "Base Asset". Even though it is mentioned there that it is sometimes referred to as Source Asset, it may be confusing to find two different wordings in the same document.

**Comment by vanlicht** [comment]
Would it then make sense to change all to "Base Asset" to be consistent?

**Comment by FelixDGG** [comment]
Yes, I believe that (Base Asset) was the agreement for the term initially, and it is mentioned in the document too, when introducing the term
